### ISSUE TITLE
Quota management for specific sites

### ISSUE THREAD

**Issue by rushideshmukh0609** [diladele/websafety-issues#782]
Hi,
We have requirement to have quota management for specific sites.
Additionally Quota should be controllered via cascaded proxy servers i.e. We will have 1 proxy in Inner DMZ and 2nd in outer DMZ and outer DMZ will be primary proxy server and and Inner DMZ server will be the child proxy servers.

**Comment by ra-at-diladele-com** [comment]
hello, Is it like - facebook should be acccessible to specific group of users (policy) with total download speed of 100kb/sec? Or something different?

**Comment by rushideshmukh0609** [comment]
Yes you are correct but this settings should be applied to parent proxy and subsequent proxy should also apply this settings.
